<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendMessageDialog</class>
 <widget class="QDialog" name="FriendMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>好友信息</string>
  </property>
  <widget class="QPushButton" name="delete_friend_Btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除好友</string>
   </property>
  </widget>
  <widget class="QLabel" name="friend_name_Label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>friend_name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_friend_note_Btn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>修改备注</string>
   </property>
  </widget>
  <widget class="QToolButton" name="headBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="noteEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>回到对话界面</string>
   </property>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
